src/main.cpp

```diff
@@ -692,7 +691,7 @@ unsigned int GetP2SHSigOpCount(const CTransaction& tx, const CCoinsViewCache& in
 
 
 
-bool CheckTransaction(const CTransaction& tx, CValidationState &state)
+bool CheckTransaction(const CTransaction& tx, CValidationState &state, uint64_t nMaxTxSize)
```

Comment:
@dcousens 
> Why not just pass consensusParams as you have done for the other two functions?
> Might as well be consistent.

This is my preference as well. Specially given that almost the other Consensus functions are going to take Consensus::Params as well, see https://github.com/bitcoin/bitcoin/pull/6591/files#diff-cbe22f30d7e480617350ef6ceca97d0cR25

I would also use ```std::numeric_limits<uint64_t>::max()``` instead of ```-1``` in the tests.